ui/app/templates/components/select-dropdown.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="field">
```

Comment:
Should we remove this and have `classNames` include `field` on the component, or do we need the extra `div`?